DSSE/environment/coverage_env.py

```diff
@@ -44,16 +55,35 @@ def __init__(
         self.all_states = {
             (x, y) for x in range(self.grid_size) for y in range(self.grid_size)
         }
+        self.repeated_coverage = 0
+        self.cumm_pos = 0
 
     def reset(self, seed=None, options=None):
-        obs, infos = super().reset(seed=seed, options=options)
+        obs, _ = super().reset(seed=seed, options=options)
+
+        self.reset_search_state()
+
+        self.cumm_pos = 0
```

Comment:
What exactly is the purpose of `cumm_pos`? Will it be used by the neural network or some RL algorithm?

Reply:
Its the success metric used in most of the papers regarding SAR missions. Can be used to compare the efficiency of different algorithms.

Reply:
That's interesting! Thank you for explaining!